src/frontend/src/lib/components/send/SendTokensList.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+	import { BigNumber } from '@ethersproject/bignumber';
+	import Listener from '$lib/components/core/Listener.svelte';
+	import TokensSkeletons from '$lib/components/tokens/TokensSkeletons.svelte';
+	import { enabledNetworkTokensUi } from '$lib/derived/network-tokens.derived';
+	import type { TokenUi } from '$lib/types/token';
+	import { fade } from 'svelte/transition';
+	import { formatToken } from '$lib/utils/format.utils';
+	import CardAmount from '$lib/components/ui/CardAmount.svelte';
+	import ExchangeTokenValue from '$lib/components/exchange/ExchangeTokenValue.svelte';
+	import { balancesStore } from '$lib/stores/balances.store';
+	import { createEventDispatcher } from 'svelte';
+	import TokenCardWithOnClick from '$lib/components/tokens/TokenCardWithOnClick.svelte';
+
+	const dispatch = createEventDispatcher();
+
+	let tokens: TokenUi[];
+	$: tokens = $enabledNetworkTokensUi.filter(({ id: tokenId }) =>
+		($balancesStore?.[tokenId]?.data ?? BigNumber.from(0n)).gt(0n)
+	);
+</script>
+
+<TokensSkeletons>
+	{#each tokens as token (token.id)}
+		<Listener {token}>
```

Comment:
Why do we need the redeclare the `Listener` here?

Reply:
I misunderstood the usage of `Listener`. You are right, it would just duplicate the Listener, since it will always be initiated together with the one in `TokensSignedIn`.

Removing it